ERC‑TBD: Minimal Avatar Smart Wallet (MASW)

MostafaS opened the pull request:
### What
Adds a new ERC draft defining the Minimal Avatar Smart Wallet (MASW), a delegate‑wallet for EIP‑7702.

### Files added
* `ERCS/eip-0.md`
* `assets/eip-0/MASW.sol`

### Discussion
https://ethereum-magicians.org/t/erc-tbd-minimal-avatar-smart-wallet-masw-delegate-wallet-for-eip-7702/24761

### Reference implementation
Relative link: [MASW.sol](assets/eip-0/MASW.sol)

### Checklist
- [x] Passes `eipw`
- [x] Uses CC0 licence line
- [x] Author field includes my GitHub handle

SamWilsn commented on ERCS/erc-7988.md:
```diff
@@ -0,0 +1,165 @@
+type: Standards Track
+category: ERC
+created: 2025-07-08
+---
```
```suggestion
requires: 7702
---
```

SamWilsn commented on ERCS/erc-7988.md:
````diff
@@ -0,0 +1,165 @@
+}
+```
+
+- A module **may** veto by reverting _or_ by returning `false`.
````
Please do use the uppercase keywords ("MAY" instead of "\*may\*").

MostafaS commented:
Hi @yedfree — MASW (ERC-7988) passes CI and includes spec, rationale, security considerations, reference implementation, and a Magicians “discussions-to” thread. There are no known open issues; could you please provide the final editorial ✅ so this can be merged as Draft? Happy to address any blockers immediately. Thanks!

MostafaS commented:
Hi @SamWilsn @xinbenlv @g11tech,

This PR has been ready for review for a while now. The Auto Review Bot is requesting an approval from one of you to move forward.

@SamWilsn - thank you for your earlier feedback in August! I addressed all your comments. Would you be able to give this an official approval review?

I'd really appreciate any feedback or approval so we can move this ERC forward.

Thank you!
